server/routes/import_wizard/html.py

```diff
@@ -30,3 +31,58 @@ def main():
 @bp.route('/')
 def main_new():
   return render_template('/import_wizard2.html')
+
+
+# This is only used in CSV + SQL backend, converting raw CSV data into resolved
+# CSV.
+@bp.route('/simple/convert', methods=['POST'])
+def convert():
+  RAW_DATA_PATH = os.environ.get("RAW_DATA_PATH")
+  SQL_DATA_PATH = os.environ.get("SQL_DATA_PATH")
+  command1 = [
+      "python",
+      "import/simple/stats/main.py",
+      "--input_path",
+      f"{RAW_DATA_PATH}/input.csv",  # TODO: use gcs and local folder path.
+      "--output_dir",
+      f"{SQL_DATA_PATH}",
+  ]
+  # TODO: save debug information in another folder
+  command2 = ["rm", f"{SQL_DATA_PATH}/debug_resolve.csv"]
+  output = []
+  for command in [command1, command2]:
+    try:
+      result = subprocess.run(command, capture_output=True, text=True)
```

Comment:
Is this blocking? Any thoughts on how this would work for longer running jobs?

Reply:
Yes, that can use "subprocess.Popen". We should definitely update to use that once there is way to check job status.